noted/accounts/v1/accounts.proto

```diff
@@ -207,3 +215,14 @@ message UpdateAccountPasswordRequest {
 message UpdateAccountPasswordResponse {
     Account account = 1 [(google.api.field_behavior) = REQUIRED];
 }
+
+
+message AuthenticateGoogleRequest {
+    string code = 1 [(google.api.field_behavior) = REQUIRED];
+}
+
+message AuthenticateGoogleResponse {
```

Comment:
Why do you return the informations of the account ? What about the jwtoken the client will need ? Look at the `AuthenticateResponse`

Reply:
Thanks, I changed with a token